src/components/clusters/wizards/osd/ClusterSettings/CloudProvider/GcpByocFields/GcpByocFields.tsx

```diff
@@ -74,7 +83,7 @@ export const GcpByocFields = (props: GcpByocFieldsProps) => {
 
   const authButtons = (
     <ToggleGroup aria-label="Authentication type">
-      {isWifDefaultEnabled ? (
+      {isWifDefaultEnabled || isOSDFromGoogleCloud ? (
```

Comment:
I think you also have to do the same thing when setting the default value for auth type here
https://github.com/zherman0/uhc-portal/blob/0f2f4eadf7ac9539be10dd94ae6a8a3550357ce9/src/components/clusters/wizards/osd/CreateOsdWizard.tsx#L342